BE/src/main/java/baseball/controller/GameController.java

```diff
@@ -0,0 +1,35 @@
+package baseball.controller;
+
+import baseball.service.GameService;
+import baseball.service.dto.GameDTO;
+import baseball.service.dto.GameScoreDTO;
+import baseball.service.dto.RequestScoreDTO;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/games")
+public class GameController {
+
+    private GameService gameService;
+
+    public GameController(GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @GetMapping
+    public List<GameDTO> showGames() {
+        return gameService.getGameDTOList();
+    }
+
+    @PostMapping("/{teamId}/score")
+    public void saveScore(@PathVariable Long teamId, @RequestBody RequestScoreDTO scoreDTO) {
+        gameService.saveScore(teamId, scoreDTO.getInningNumber(), scoreDTO.getScore());
```

Comment:
```suggestion
        gameService.saveScore(teamId, scoreDTO);
```
getter는 최대한 사용하지 않고, 로직을 진행하면 어떨까요?

Reply:
넵 RequestScoreDTO는 ScoreRequest로 이름 변경하였고, 해당 타입 변수만 매개변수로 받도록 수정했습니다!